source/server/server.cc

```diff
@@ -300,19 +300,19 @@ RunHelper::RunHelper(Event::Dispatcher& dispatcher, Upstream::ClusterManager& cm
 
   // Setup signals.
   sigterm_ = dispatcher.listenForSignal(SIGTERM, [this, &hot_restart, &dispatcher]() {
-    ENVOY_LOG(warn, "caught SIGTERM");
```

Comment:
I think these warning should show up at warn level, as they are exceptional events.

Reply:
done